tests/unit/DistributionTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * This file is part of graze/dog-statsd
+ *
+ * Copyright (c) 2016 Nature Delivered Ltd. <https://www.graze.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license https://github.com/graze/dog-statsd/blob/master/LICENSE.md
+ * @link    https://github.com/graze/dog-statsd
+ */
+
+namespace Graze\DogStatsD\Test\Unit;
+
+use Graze\DogStatsD\Test\TestCase;
+
+class DistributionTest extends TestCase
+{
+    public function testDistribution()
+    {
+        $this->client->distribution('test_metric', 10);
+        $this->assertEquals('test_metric:10|d', $this->client->getLastMessage());
+
+        $this->client->distribution('test_metric', 1.2);
+        $this->assertEquals('test_metric:1.2|d', $this->client->getLastMessage());
+    }
+
+    public function testDistributionSample()
+    {
+        while ($this->client->getLastMessage() === '') {
+            $this->client->distribution('test_metric', 5, 0.75);
+        }
+        $this->assertEquals('test_metric:5|d|@0.75', $this->client->getLastMessage());
+    }
+
+    public function testIncrementSampleFailure()
```

Comment:
```suggestion
    public function testDistributionSampleFailure()
```

Reply:
I've pushed the requested change, sorry for missing it